code/modules/mob/living/living.dm

```diff
@@ -950,10 +950,6 @@
 			var/mob/living/L = pulledby
 			L.set_pull_offsets(src, pulledby.grab_state)
 
-//	if(active_storage && !(CanReach(active_storage.parent,view_only = TRUE)))
-	if(active_storage)
```

Comment:
add an override var for bins rather than deleting this line. This functionality could be used elsewhere.